openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -0,0 +1,218 @@
+package node
```

Comment:
Please update the PR so that I can see the diff applied to this file and the others. That will make it easier to review further.

Reply:
I believe you can in https://github.com/openshift/kubernetes/pull/2201/commits/92080d301e6bd7900d1fd8dae2825037ba5930b0 and https://github.com/openshift/kubernetes/pull/2201/commits/99174f98930e109b23e16e6622558f7a01a7cf22  as https://github.com/openshift/kubernetes/pull/2201/commits/95bf4b8aa869daf76cade9ae4e7749eb912cdf17 is just a simple copy